### PR TITLE
Install and configure Redux dev tools

### LINKED ISSUE #22: 10/6 tasks
Made our first team rebase
Got our redux store setup w/ Dev Tools
Completed our first solo spike looking at the code
BE/FE breakout to look through the code
Visualization of current schema
General annotations of what most of the files are
Added notes to packages / installs / imports
Made notes on usability / flow of the site 
Made general notes of how we would implement the desired features on our track

### PR DESCRIPTION
**Pull request by ajtran303**
This commit installs and configures the Redux dev tools package.

It also updates all the dependencies in the project.

### CODE REVIEW

**Comment by leighlars** [comment]
Awesome thanks for getting this done!

**Comment by kip-west** [comment]
Great work! Glad we're hooked up to the Chrome Dev Tools.

**Review by kip-west — APPROVED** [review]

**Comment by carlymclift** [comment]
Thanks for doing this

File: code/api/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.9.2",
-    "bcrypt": "4.0.1",
+    "bcrypt": "^5.0.0",
```
Comment (kathytbui) [inline code/api/package.json:30]: Bcrypt for life! Glad this takes care of everything

**Review by kathytbui — APPROVED** [review]

**Pull request merged**